Update docs theme

raphink opened the pull request:
Signed-off-by: Raphaël Pinson <raphael@isovalent.com>

raphink committed:
Update docs theme

Signed-off-by: Raphaël Pinson <raphael@isovalent.com>

qmonnet reviewed (changes requested):
I forgot that the branch had to be opened in the Cilium repo to have the automated update with the new `docs-builder` image in the PR. I'll update manually tomorrow.

qmonnet committed:
ci: update docs-builder

Manual update for docs-builder tag, because the related PR's branch was
not opened in the Cilium repository (but from a fork).

Signed-off-by: Quentin Monnet <quentin@isovalent.com>
